components/Header.js

```diff
@@ -10,24 +10,67 @@ class Header extends Component {
     super(props);
 
     this.state = { menuHidden: true };
+    this.aboutMenu = React.createRef();
+    this.handleEventOutsideAboutMenu = this.handleEventOutsideAboutMenu.bind(this);
   }
 
-  componentDidUpdate(prevProps, prevState, snapshot) {
-    if (this.state.menuHidden !== prevState.menuHidden) {
-      this.props.onMenuToggle(this.state.menuHidden);
-    }
+  componentDidMount() {
+    document.addEventListener('focusin', this.handleEventOutsideAboutMenu);
+    document.addEventListener('mousedown', this.handleEventOutsideAboutMenu);
+  }
+
+  componentWillUnmount() {
+    document.removeEventListener('focusin', this.handleEventOutsideAboutMenu);
+    document.removeEventListener('mousedown', this.handleEventOutsideAboutMenu);
   }
 
-  handleMenuToggle = e => {
-    // console.log(e.target);
-    if (window.innerWidth <= parseInt(this.props.theme.medium)) {
+  handleEventOutsideAboutMenu = ({ target }) => {
+    if (!this.aboutMenu) return;
+    if (target === this.aboutMenu.current || this.aboutMenu.current.contains(target)) return;
+    this.hideAboutMenu();
+  };
+
+  hideMenu = () => {
+    this.setState(() => ({
+      menuHidden: true,
+    }));
+  };
+
+  showAboutMenu = () => {
+    if (!this.aboutMenu) return;
+    this.aboutMenu.current.setAttribute('open', 'open');
+  };
+
+  hideAboutMenu = () => {
+    if (!this.aboutMenu) return;
+    this.aboutMenu.current.removeAttribute('open');
+  };
+
+  handleMenuLinkClick = () => {
+    this.hideMenu();
+    this.hideAboutMenu();
+  };
+
+  handleMenuLinkKeyDown = e => {
+    if (e.key === 'Escape') {
+      this.hideMenu();
+      this.hideAboutMenu();
+    }
+  };
+
+  handleMenuToggle = () => {
+    const { theme } = this.props;
```

Comment:
No meaningful change to this this function -- just used destructuring so that we could remove that linter ignore statement from the top of this file.